### DISCUSSION TITLE
Create a wind particle animation

### DISCUSSION THREAD

**Discussion by MahmoudMousaHamad** [rnmapbox/maps#3588] [Q&A]
I have a react native application that uses mapbox for rendering a map. I would like to add a layer for wind using the react native mapbox SDK (and not a web view). I am trying to achieve a similar effect to the one implemented here: https://docs.mapbox.com/mapbox-gl-js/example/raster-particle-layer/

I tried using a web view, but it is not ideal as we are already using mapbox react native SDK for the entire app.

**Comment by mfazekas** [comment]
See #3582
